src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.5.6'
+__version__ = '1.6.0'
```

Comment:
```suggestion
__version__ = '1.7.0'
```
Version 1.6 was just pushed.  Bumping this to 1.7 would resolve the version conflict and possibly restart the travis validation.